.github/workflows/ci.yml

```diff
@@ -10,10 +10,16 @@ name: ci
 jobs:
   lint-unit:
     uses: sous-chefs/.github/.github/workflows/lint-unit.yml@3.1.1
+    permissions:
+      actions: write
+      checks: write
+      pull-requests: write
+      statuses: write
+      issues: write
 
   integration:
     needs: lint-unit
-    runs-on: macos-10.15
+    runs-on: ubuntu-latest
```

Comment:
under env: in the steps section need to add `KITCHEN_LOCAL_YAML: kitchen.dokken.yml`